integrations/cdi/datasource-ucp/pom.xml

```diff
@@ -75,17 +70,23 @@
 
         <!-- Runtime-scoped dependencies. -->
         <dependency>
-            <groupId>io.smallrye</groupId>
-            <artifactId>jandex</artifactId>
+            <groupId>io.helidon.integrations.db</groupId>
+            <artifactId>ojdbc</artifactId>
```

Comment:
The prior version had our `ojdbc` integration artifact brought in at `compile` time, which is not necessary (only `ucp11` is necessary at compile time). That means, too, that perhaps this dependency is not needed? I should say: it _definitely_ is not needed but I seem to recall others having strong opinions here, i.e. that pulling in the complete suite of Oracle production-level `ojdbc`-related artifacts at `runtime` is a convenient and proper thing to do. Personally I would vote to remove this dependency entirely and put the onus on the user to include it if she wants. What say others, particularly @arjav-desai  and @barchetta?

Reply:
Sounds reasonable.